### ISSUE TITLE
raco setup seems to be rebuilding docs even though nothing changed

### ISSUE THREAD

**Issue by lexi-lambda** [racket/racket#1401]
Running on HEAD, `raco setup` seems to be having some trouble with scribble docs, but I’m not entirely sure what’s wrong. I nuked my local copy and reinstalled everything, and I’m still getting the problem. Here are the steps I took to reproduce the issue, starting from a fresh clone of the repository:

```
make PKGS=
raco pkg install --auto xrepl
raco pkg install --auto drracket
```

When installing `drracket`, the setup step created a lot of noise when rendering the docs:

<details>
<summary>**`raco pkg` output**</summary>
```
raco setup: WARNING: failed to find info for path: (doc compatibility out1.sxref)
raco setup: WARNING: failed to find info for path: (doc errortrace out1.sxref)
raco setup: WARNING: failed to find info for path: (doc future-visualizer out0.sxref)
raco setup: WARNING: failed to find info for path: (doc future-visualizer out1.sxref)
raco setup: WARNING: failed to find info for path: (doc net out1.sxref)
raco setup: WARNING: failed to find info for path: (doc r5rs out0.sxref)
raco setup: WARNING: failed to find info for path: (doc r6rs out0.sxref)
raco setup: WARNING: failed to find info for path: (doc xrepl out0.sxref)
raco setup: WARNING: failed to find info for path: (doc syntax out1.sxref)
raco setup: WARNING: failed to find info for path: (doc net out1.sxref)
raco setup: WARNING: failed to find info for path: (doc continue out0.sxref)
raco setup: WARNING: failed to find info for path: (doc syntax out1.sxref)
raco setup: WARNING: failed to find info for path: (doc errortrace out1.sxref)
raco setup: WARNING: failed to find info for path: (doc file out1.sxref)
raco setup: WARNING: failed to find info for path: (doc mzscheme out0.sxref)
raco setup: WARNING: failed to find info for path: (doc mzlib out1.sxref)
raco setup: WARNING: failed to find info for path: (doc planet out1.sxref)
raco setup: WARNING: failed to find info for path: (doc profile out1.sxref)
raco setup: WARNING: failed to find info for path: (doc r5rs out1.sxref)
raco setup: WARNING: failed to find info for path: (doc pict out1.sxref)
raco setup: WARNING: failed to find info for path: (doc net out1.sxref)
raco setup: WARNING: failed to find info for path: (doc file out1.sxref)
raco setup: WARNING: failed to find info for path: (doc pict out1.sxref)
raco setup: WARNING: failed to find info for path: (doc mrlib out1.sxref)
raco setup: WARNING: failed to find info for path: (doc string-constants out1.sxref)
raco setup: WARNING: failed to find info for path: (doc syntax-color out1.sxref)
raco setup: WARNING: failed to find info for path: (doc xml out1.sxref)
raco setup: WARNING: failed to find info for path: (doc continue out0.sxref)
raco setup: WARNING: failed to find info for path: (doc net out1.sxref)
raco setup: WARNING: failed to find info for path: (doc readline out1.sxref)
raco setup: WARNING: failed to find info for path: (doc web-server out0.sxref)
raco setup: WARNING: failed to find info for path: (doc xrepl out0.sxref)
raco setup: WARNING: failed to find info for path: (doc xrepl out1.sxref)
raco setup: WARNING: failed to find info for path: (doc xml out0.sxref)
raco setup: WARNING: failed to find info for path: (doc xml out1.sxref)
raco setup: WARNING: failed to find info for path: (doc openssl out1.sxref)
raco setup: WARNING: failed to find info for path: (doc planet out0.sxref)
raco setup: WARNING: failed to find info for path: (doc net out1.sxref)
raco setup: WARNING: failed to find info for path: (doc pict out1.sxref)
raco setup: WARNING: failed to find info for path: (doc syntax out1.sxref)
raco setup: WARNING: failed to find info for path: (doc future-visualizer out0.sxref)
raco setup: WARNING: failed to find info for path: (doc parser-tools out1.sxref)
raco setup: WARNING: failed to find info for path: (doc planet out0.sxref)
raco setup: WARNING: failed to find info for path: (doc planet out1.sxref)
raco setup: WARNING: failed to find info for path: (doc r5rs out0.sxref)
raco setup: WARNING: failed to find info for path: (doc r5rs out1.sxref)
raco setup: WARNING: failed to find info for path: (doc r6rs out0.sxref)
raco setup: WARNING: failed to find info for path: (doc r6rs out1.sxref)
raco setup: WARNING: failed to find info for path: (doc readline out1.sxref)
raco setup: WARNING: failed to find info for path: (doc rackunit out1.sxref)
raco setup: WARNING: failed to find info for path: (doc syntax-color out1.sxref)
raco setup: WARNING: failed to find info for path: (doc ts-guide out0.sxref)
raco setup: WARNING: failed to find info for path: (doc pict out0.sxref)
raco setup: WARNING: failed to find info for path: (doc web-server out0.sxref)
raco setup: WARNING: failed to find info for path: (doc xrepl out0.sxref)
raco setup: WARNING: failed to find info for path: (doc xrepl out1.sxref)
raco setup: WARNING: failed to find info for path: (doc syntax out1.sxref)
raco setup: WARNING: failed to find info for path: (doc ts-reference out0.sxref)
raco setup: WARNING: failed to find info for path: (doc continue out0.sxref)
raco setup: WARNING: failed to find info for path: (doc pict out1.sxref)
raco setup: WARNING: failed to find info for path: (doc file out1.sxref)
raco setup: WARNING: failed to find info for path: (doc mrlib out1.sxref)
raco setup: WARNING: failed to find info for path: (doc xrepl out0.sxref)
raco setup: WARNING: failed to find info for path: (doc compatibility out1.sxref)
raco setup: WARNING: failed to find info for path: (doc mzlib out1.sxref)
raco setup: WARNING: failed to find info for path: (doc compatibility out0.sxref)
raco setup: WARNING: failed to find info for path: (doc db out0.sxref)
raco setup: WARNING: failed to find info for path: (doc dynext out0.sxref)
raco setup: WARNING: failed to find info for path: (doc errortrace out0.sxref)
raco setup: WARNING: failed to find info for path: (doc embedded-gui out0.sxref)
raco setup: WARNING: failed to find info for path: (doc file out0.sxref)
raco setup: WARNING: failed to find info for path: (doc continue out0.sxref)
raco setup: WARNING: failed to find info for path: (doc future-visualizer out0.sxref)
raco setup: WARNING: failed to find info for path: (doc json out0.sxref)
raco setup: WARNING: failed to find info for path: (doc data out0.sxref)
raco setup: WARNING: failed to find info for path: (doc mrlib out0.sxref)
raco setup: WARNING: failed to find info for path: (doc mzscheme out0.sxref)
raco setup: WARNING: failed to find info for path: (doc osx-ssl out0.sxref)
raco setup: WARNING: failed to find info for path: (doc openssl out0.sxref)
raco setup: WARNING: failed to find info for path: (doc mzlib out0.sxref)
raco setup: WARNING: failed to find info for path: (doc parser-tools out0.sxref)
raco setup: WARNING: failed to find info for path: (doc planet out0.sxref)
raco setup: WARNING: failed to find info for path: (doc net out0.sxref)
raco setup: WARNING: failed to find info for path: (doc profile out0.sxref)
raco setup: WARNING: failed to find info for path: (doc r5rs out0.sxref)
raco setup: WARNING: failed to find info for path: (doc r6rs out0.sxref)
raco setup: WARNING: failed to find info for path: (doc readline out0.sxref)
raco setup: WARNING: failed to find info for path: (doc rackunit out0.sxref)
raco setup: WARNING: failed to find info for path: (doc scriblib out0.sxref)
raco setup: WARNING: failed to find info for path: (doc srfi out0.sxref)
raco setup: WARNING: failed to find info for path: (doc string-constants out0.sxref)
raco setup: WARNING: failed to find info for path: (doc syntax-color out0.sxref)
raco setup: WARNING: failed to find info for path: (doc ts-guide out0.sxref)
raco setup: WARNING: failed to find info for path: (doc web-server-internal out0.sxref)
raco setup: WARNING: failed to find info for path: (doc pict out0.sxref)
raco setup: WARNING: failed to find info for path: (doc win32-ssl out0.sxref)
raco setup: WARNING: failed to find info for path: (doc web-server out0.sxref)
raco setup: WARNING: failed to find info for path: (doc xrepl out0.sxref)
raco setup: WARNING: failed to find info for path: (doc xml out0.sxref)
raco setup: WARNING: failed to find info for path: (doc syntax out0.sxref)
raco setup: WARNING: failed to find info for path: (doc ts-reference out0.sxref)
raco setup: WARNING: failed to find info for path: (doc plot out0.sxref)
raco setup: WARNING: failed to find info for path: (doc math out0.sxref)
raco setup: WARNING: failed to find info for path: (doc data out1.sxref)
raco setup: WARNING: failed to find info for path: (doc rackunit out1.sxref)
raco setup: WARNING: failed to find info for path: (doc xml out0.sxref)
raco setup: WARNING: failed to find info for path: (doc pict out0.sxref)
raco setup: WARNING: failed to find info for path: (doc pict out1.sxref)
raco setup: WARNING: failed to find info for path: (doc dynext out1.sxref)
raco setup: WARNING: failed to find info for path: (doc planet out0.sxref)
raco setup: WARNING: failed to find info for path: (doc rackunit out1.sxref)
raco setup: WARNING: failed to find info for path: (doc xml out1.sxref)
raco setup: WARNING: failed to find info for path: (doc syntax out1.sxref)
raco setup: WARNING: failed to find info for path: (doc file out1.sxref)
raco setup: WARNING: failed to find info for path: (doc mzscheme out0.sxref)
raco setup: WARNING: failed to find info for path: (doc net out1.sxref)
raco setup: WARNING: failed to find info for path: (doc scriblib out1.sxref)
raco setup: WARNING: failed to find info for path: (doc pict out0.sxref)
raco setup: WARNING: failed to find info for path: (doc pict out1.sxref)
raco setup: WARNING: failed to find info for path: (doc xml out1.sxref)
raco setup: WARNING: failed to find info for path: (doc syntax out1.sxref)
raco setup: WARNING: failed to find info for path: (doc ts-reference out0.sxref)
raco setup: WARNING: failed to find info for path: (doc errortrace out1.sxref)
raco setup: WARNING: failed to find info for path: (doc mrlib out1.sxref)
raco setup: WARNING: failed to find info for path: (doc mzlib out1.sxref)
raco setup: WARNING: failed to find info for path: (doc planet out1.sxref)
raco setup: WARNING: failed to find info for path: (doc net out1.sxref)
raco setup: WARNING: failed to find info for path: (doc string-constants out1.sxref)
raco setup: WARNING: failed to find info for path: (doc pict out1.sxref)
raco setup: WARNING: failed to find info for path: (doc syntax out1.sxref)
```
</details>

Subsequently, running `raco setup` does not produce the above warning messages, but it *does* cause all the documentation to be rebuilt, even though nothing has changed.

I am running OS X 10.11.4, if that affects anything.

**Comment by lexi-lambda** [comment]
This issue appears to have been introduced by fc345ed249c7393bce77b31f5f5812d487ef6958, given that I cannot reproduce it before that commit.

**Comment by lexi-lambda** [comment]
Fixed by e3ae0103af8b152d37d19a36f6ec15abb3887280.
